Rain World

LeadOnTaste opened the pull request:
<!--
**Это перевод оригинального темплейта с Baystation12/Baystation12**
Не забудьте добавить ченджлог, если вы сделали изменения, касающиеся админов/игроков, и которые могут повлиять на геймплей.

Примеры, когда необходимо добавить ченджлог:
* Добавление/удаление объектов с которыми игроки могут взаимодействовать, или их поведение.
* Добавление/удаление/изменение админских инструментов/кнопок.
* Изменения карты.

Примеры, когда записи в ченджлоге необязательны/не требуются:
* Косметические изменения, такие как описания, звуки и т.д.
* Оптимизация и другие подобные изменения базовых систем, не влияющие на геймплей.
* Небольшие багфиксы.

Вы можете найти README файл с подробными объяснениями в ./html/changelogs.
А ещё можно добавить свой ченджлог прямо в ПР. Инфа тут: https://github.com/Proxima-Project/Proxi-Bay12/wiki/Automatic-changelog-generation
-->
### Описание изменений
* Добавлена слизнекошь Newt. Просто очередной рескин кошки, который позже станет всеядным.
<!--
Здесь вы пишите, какой прекрасный йогурт вы сделали.
-->
### Почему и что этот ПР улучшит
Котики.
<!--
Здесь вы пишите, почему ваш йогурт лучше чем тот, что был.
-->
### Авторство
LeadOnTaste
<!--
Здесь вы пишите, если этот йогурт был украден у сырков.
-->
### Чейнджлог
* Добавлена слизнекошь.
<!--
Здесь вы пишите, если о новом йогурте должны узнать игроки. Пример смотрите в шапке
-->
#### Требуемые изменения на википедии проекта
* Вписать в статью по мобам.
<!--
Здесь требуется описание всех изменений, что могут, потенциально, затрагивать лор или геймплейную составляющую игры, ввиду чего требуется обновление википедии.
Если вы не уверены в том, требуются ли они и какие именно - обратитесь к Директору Редакторума в нашем Дискорд сервере (https://discord.gg/F7GgD86Pvf)
Вот наша основная вики: https://wiki.proxima.fun/wiki/Заглавная_страница
-->

LeadOnTaste committed:
Slugcat Code

Добавил код Ньют. Предполагается возможность появления других особей вида Felis Limax.

LeadOnTaste committed:
Slugcat Sprites

Добавил спрайты слизнекота.

johncoder0 commented:
я не кодер проксимы но

ты насрал кодом и иконками в папки бея, а не модули проксимы. 
сам потом будешь конфликты при мерже с беем решать

polerno commented:
> я не кодер проксимы но
> 
> ты насрал кодом и иконками в папки бея, а не модули проксимы. сам потом будешь конфликты при мерже с беем решать

Хоть когда-то ты сказал умные вещи

johncoder0 commented:
а слизнекот прикольный одобряю

polerno commented on code/modules/mob/living/simple_animal/friendly/cat.dm:
```diff
@@ -214,6 +214,28 @@
+	name = "Newt"
+	desc = "That's a cat-like slug creature, found on a rainy exoplanet of VD03CT and adopted by Expeditionary Corps just to be soon kidnapped by Fleet. Wawawawawawawa."
+	gender = FEMALE
+	icon_state = "slugcat"
```
Пожалуйста не трогай файлы бэя, мы не хотим плакать при мерже

LeadOnTaste replied:
если я буду кидать код не в файлы бея, оно может наебнуться, да и спрайты у всех мобов в одном файле в отличие от тг, так что оно так не поделится

polerno replied:
> если я буду кидать код не в файлы бея, оно может наебнуться

Кто не рискует, тот не пьёт шампанского и не ебёт женщин

LeadOnTaste replied:
я тупо не смогу разделить спрайты

FeenieRU replied:
Флаффы не сломаются, если ты делаешь наследуемый в другом файле.

polerno replied:
> я тупо не смогу разделить спрайты

Не сломается, всё заебись будет

johncoder0 replied:
просто добавь дми со слизнекотом в папку проксимы и укажи в коде путь к новым иконкам

LeadOnTaste replied:
Я хз как оно у симплмобов вообще выглядит и потому ссусь это делать, ибо не ебу где вообще у них прописана строчка спрайта.

johncoder0 replied:
везде и всегда используется переменная icon
`icon = 'icons/mob/simple_animal/pizdotryas.dmi'`

LeadOnTaste replied:
ок, закончил, но просьба потом вынести за меня отдельно в подпапку для симплмобов это дело, я хз как их в гитхабе самом делать

polerno commented on code/modules/mob/living/simple_animal/friendly/cat.dm:
```diff
@@ -214,6 +214,28 @@
+	maxbodytemp = 395 // Above 121 Degree Celsius. Yes, hot bath for a slugcat.
+	holder_type = /obj/item/holder/cat/fluff/newt
+	say_list_type = /datum/say_list/cat/newt
+	var/friend_name = pick("Damien Hawkins",
```
ЛАДНО, ТЫ ЖЕ ГОВОРИЛ ЧТО ЭТО ОСОБЬ ЖЕНСКОГО ПОЛА. НАХУЯ ЕЙ МУЖСКИЕ ИМЕНА

johncoder0 replied:
это список раундстартовых друзей животного, оно будет смешно тереться об хозяев вроде.
дамьен - персонаж тейсти

LeadOnTaste replied:
Это имена тех, кого кошь считает по дефолту рабом. Тут оно будет пикать рандомно либо меня, либо тебя, либо Еву.

polerno replied:
> Это имена тех, кого кошь считает по дефолту рабом. Тут оно будет пикать рандомно либо меня, либо тебя, либо Еву.

ладно, тогда оставь

johncoder0 replied:
(еллонкейк на секунду забыл что такое пик)

LeadOnTaste replied:
> это список раундстартовых друзей животного, оно будет смешно тереться об хозяев вроде. дамьен - персонаж тейсти

и вообще Дэмиэн, заебали коверкать... И Я ХОКИНС А НЕ ХОУКИНС ИЛИ ХАВКИНС СУКА!

polerno commented on code/modules/mob/living/simple_animal/friendly/cat.dm:
```diff
@@ -214,6 +214,28 @@
 
+/obj/item/holder/cat/fluff/newt
+	name = "Newt"
+	desc = "It's Newt! Wawawawawawawa."
```
WAWAWAWAWAWAWAWA

polerno replied:
Меня это угарнула, так что это можно оставить

polerno commented:
Если это не добавят, я взорву кваритры хед.состава
![image](https://user-images.githubusercontent.com/48944552/230344557-e81ce64a-f76a-4dd6-b68a-d2ddd09c8f55.png)

LeadOnTaste committed:
Newt Code Reloc Part 1

FeenieRU reviewed (changes requested):
Требуется перенос под модуляр.

polerno commented:
Пойду спать, пинганёте как он всё сделает

LeadOnTaste committed:
Slugcat Code Reloc Part 2

Передвинул код слизнекоши и добавил ноунейм версию. Теперь можно будет делать много разных слизнекотов.

LeadOnTaste committed:
Slugcat Sprite Reloc Part 2

LeadOnTaste committed:
Slugcat Sprites

LeadOnTaste committed:
Slugcat Sprites Reloc Part 1

LeadOnTaste committed:
Slugcat Code Fix

срезал лишнюю строчку

LeadOnTaste committed:
Code Correction

Подформатировал для ровности.

LeadOnTaste commented:
Я ВСЁ НАХУЙ

LeadOnTaste commented:
@polerno @FeenieRU готово

polerno reviewed (approved):
ВОТ ТЕПЕРЬ ТЫ СДЕЛАЛ ЭТО НОРМАЛЬНО

FeenieRU reviewed (approved):
Перенос совершён.

Evie7056 commented:
Сука, оно автозакрылось от моего имени-

LeadOnTaste commented:
> Сука, оно автозакрылось от моего имени-

не ссы там нормально код
